### PR TITLE
chore: use ffmpeg bundled with Playwright

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: src/traceViewer/web/ui/filmStripView.ts [hunk 1]
```diff
@@ -63,9 +63,12 @@ export class FilmStripView {
 
   private async _initialize() {
     for (const video of this._videos) {
+      debugger;
```
Comment (mxschmitt) [inline src/traceViewer/web/ui/filmStripView.ts:66]: missing debugger statement

File: src/traceViewer/videoTileGenerator.ts [hunk 2]
```diff
@@ -30,13 +30,24 @@ export class VideoTileGenerator {
   }
 
   async render(events: PageVideoTraceEvent[]) {
+    let ffmpegName = '';
+    if (process.platform === 'win32') {
+      if (process.arch === 'ia32')
+        ffmpegName = 'ffmpeg-win32';
```
Comment (mxschmitt) [inline src/traceViewer/videoTileGenerator.ts:36]: I remember a PSA that we drop win i32?

File: src/traceViewer/videoTileGenerator.ts [hunk 3]
```diff
@@ -30,13 +30,24 @@ export class VideoTileGenerator {
...
+      else
+        ffmpegName = 'ffmpeg-win64';
+    }
+    if (process.platform === 'darwin')
+      ffmpegName = 'ffmpeg-mac';
+    if (process.platform === 'linux')
+      ffmpegName = 'ffmpeg-linux';
+    const ffmpeg = path.join(path.dirname(require.resolve('playwright')), 'third_party', 'ffmpeg', ffmpegName);
```
Comment (mxschmitt) [inline src/traceViewer/videoTileGenerator.ts:44]: this logic looks very similar to a logic which we have upstream. Maybe we can expose it somehow internally as a util function?

**Review by mxschmitt — CHANGES_REQUESTED** [review]

File: src/traceViewer/web/ui/filmStripView.ts (hunk 1 above)
Comment (pavelfeldman) [reply src/traceViewer/web/ui/filmStripView.ts:66]: woohoo

File: src/traceViewer/videoTileGenerator.ts (hunk 2 above)
Comment (pavelfeldman) [reply src/traceViewer/videoTileGenerator.ts:36]: i don't think we made that decision

File: src/traceViewer/videoTileGenerator.ts (hunk 3 above)
Comment (pavelfeldman) [reply src/traceViewer/videoTileGenerator.ts:44]: i don't think it is worth it

File: src/traceViewer/videoTileGenerator.ts (hunk 2 above)
Comment (mxschmitt) [reply src/traceViewer/videoTileGenerator.ts:36]: ah ye was FF only: https://playwright.slack.com/archives/CSUHZPVLM/p1596215581072600

**Review by mxschmitt — APPROVED** [review]

**Pull request merged**